src/dsp_tools/commands/xmlupload/resource_create_client.py

```diff
@@ -103,35 +104,28 @@ def _make_resource(
         resource: XMLResource,
         bitstream_information: BitstreamInfo | None,
     ) -> dict[str, Any]:
-        resource_iri = resource.iri
+        migration_metadata = None
+        res_iri = resource.iri
         if resource.ark:
-            resource_iri = convert_ark_v0_to_resource_iri(resource.ark)
-        context = get_json_ld_context_for_project(self.project_onto_dict)
-        res = {
-            "@type": resource.restype,
-            "rdfs:label": resource.label,
-            "knora-api:attachedToProject": {"@id": self.project_iri},
-            "@context": context,
-        }
-        if resource_iri:
-            res["@id"] = resource_iri
-        if resource.permissions:
-            if perm := self.permissions_lookup.get(resource.permissions):
-                res["knora-api:hasPermissions"] = str(perm)
-            else:
-                raise PermissionNotExistsError(
-                    f"Could not find permissions for resource {resource.res_id} with permissions {resource.permissions}"
-                )
-        if resource.creation_date:
-            res["knora-api:creationDate"] = {
-                "@type": "xsd:dateTimeStamp",
-                "@value": str(resource.creation_date),
-            }
+            res_iri = convert_ark_v0_to_resource_iri(resource.ark)
+        if res_iri:
```

Comment:
Why is this dependent on the res_iri? It should rather be dependent on whether a creation date is existent or not:

```suggestion
        if resource.creation_date:
```